doc/devel/MEP/MEP23.rst

```diff
@@ -76,15 +76,15 @@ will add the following new methods
 --------------------
 
 To control which `FigureManager` will contain the new figures, an
-extra optional parameter `figuremanager` will be added, this parameter
+extra optional parameter *figuremanager* will be added, this parameter
 value will be passed to `new_figure_manager_given_figure`
 
 `new_figure_manager_given_figure`
 ---------------------------------
 
-* If `figuremanager` parameter is give, this `FigureManager` object
+* If *figuremanager* parameter is given, this `FigureManager` object
   will be used instead of creating a new one.
-* If `rcParams['backend.multifigure'] == True`: The last
+* If ``rcParams['backend.multifigure'] == True``: The last
```

Comment:
```suggestion
* If ``rcParams['backend.multifigure']`` is True: The last
```
``x == True`` is non-ideomatic as code expression

Reply:
done